script.lua

```diff
@@ -640,6 +641,17 @@ end
 
 setmetatable(keneanung.bashing.systems, { __index = getSystem } )
 
+keneanung.bashing.isPossibleTarget = function(targetName)
+	local prios = keneanung.bashing.configuration.priorities
+	local area = gmcp.Room.Info.area
+
+	if prios[area] == nil then
+		prios[area] = {}
+	end
+
+	return table.contains(prios[area], targetName)
+end
```

Comment:
This will create empty tables in the priority list when running a simple query. And it couples a rather generic function to a specific location in game. I'd probably change it to this:

```suggestion
keneanung.bashing.isPossibleTarget = function(targetName, area)
	if not area then
		area = gmcp.Room.Info.area
	end
	local prios = keneanung.bashing.configuration.priorities[area] or {}
	return table.contains(prios, targetName)
end
```

This way it's more generic and `area` is an optional parameter